### PR TITLE
[docs] Add example to the dissect docs

### PR DESCRIPTION
**Pull request by dedemorton**
## What does this PR do?

Adds a real-world example to the dissect docs.

## Why is it important?

Examples sometimes say more than words.

## Checklist

- [x] My code follows the style guidelines of this project

### CODE REVIEW

File: libbeat/processors/dissect/docs/dissect.asciidoc [hunk 1]
```diff
@@ -6,10 +6,10 @@ The dissect processor tokenizes incoming strings using defined patterns.
 [source,yaml]
 -------
 processors:
-- dissect:
-    tokenizer: "%{key1} %{key2}"
-    field: "message"
-    target_prefix: "dissect"
+  - dissect:
```
Comment (dedemorton) [inline libbeat/processors/dissect/docs/dissect.asciidoc]: This is a bit trivial, but in the filebeat.yml, we already have some processors defined:

```
processors:
  - add_host_metadata: ~
  - add_cloud_metadata: ~
  - add_docker_metadata: ~
  - add_kubernetes_metadata: ~
````

For copy pastability, I think we should use the same format in the docs, or change the .yml files where we define the processors. I dunno which is correct, but both work as long as you're consistent. WDYT?

Comment (urso) [reply libbeat/processors/dissect/docs/dissect.asciidoc]: There is no 'correct', but variants on how to do things. Yay YAML :)

+1 on consistency. I presume changing the reference config files would be easier?

Comment (dedemorton) [reply libbeat/processors/dissect/docs/dissect.asciidoc]: Yes changing the reference config files would *definitely* be easier. I've created an issue to track this. I mean, it's minor, but I hate when people make me do extra work. :-P

Comment (dedemorton) [reply libbeat/processors/dissect/docs/dissect.asciidoc]: https://github.com/elastic/beats/issues/16696

**Pull request merged**